
Replace 3rdparty 8723de driver by the new rtw88 driver.

Add a module alias to keep upgrade path.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- a/drivers/net/wireless/realtek/rtw88/rtw8723de.c
+++ b/drivers/net/wireless/realtek/rtw88/rtw8723de.c
@@ -28,3 +28,4 @@ module_pci_driver(rtw_8723de_driver);
 MODULE_AUTHOR("Realtek Corporation");
 MODULE_DESCRIPTION("Realtek 802.11n wireless 8723de driver");
 MODULE_LICENSE("Dual BSD/GPL");
+MODULE_ALIAS("8723de");
